src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -21,14 +23,23 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
```

Comment:
Yes it is used for logging to add context to messages, usually workflow ID and perhaps call name / index.  But here we're operating at the workflow level so there is no call context, and this is before a workflow ID has been generated.  So the tag here is just the actor name which would be the same for all workflow submissions and therefore not so useful.

This could be enhanced to try to at least pull out the workflow name, but I don't think that should be part of these changes.